integration_tests/test_global_set.py

```diff
@@ -0,0 +1,5 @@
+from lpython import i32
+
+s1: set[str] = {"a", "b", "c", "a"}
+s2: set[i32] = {1, 2, 3, 1}
+s3: set[tuple[i32, i32]] = {(1, 2), (2, 3), (4, 5)}
```

Comment:
Can you add some kind of print or assert statement(s) here? For example, we can have an assert on the size of the set.

Reply:
Sure!